lib/main.dart

```diff
@@ -48,3 +159,33 @@ final Widget drawerIcon = SvgPicture.asset(
   height: 14,
   width: 24,
 );
+
+Widget imageSlide(String? image, String? title) {
+  return Stack(
+    children: [
+      Container(
+        color: Colors.grey[100],
+        height: 195,
+        width: 714,
+        child: Image(
+          image: AssetImage(image!),
+          fit: BoxFit.none,
+        ),
+      ),
+      Positioned(
+        bottom: 70,
+        right: 13,
+        child: Text(
+          title!,
```

Comment:
null safety becomes useless in this condition.
What if the title param is null? The app will crash.
Please handle it correctly @edoliantesjr 
Avoid useing **`variable!`** display blank text if the param is null. Use `title??''` instead